public/mushrooms.json

```diff
@@ -2,7 +2,7 @@
     {
         "latin": "Amanita muscaria",
         "common": ["Fly agaric", "Fly amanita"],
-        "confusedWith": ["Amanita caesarea"],
+        "confusedWith": ["Amanita caesarea", "Amanita aprica", "Amanita gemmata", "white forms confused with Amanita alba, Amanita phalloides"],
```

Comment:
If entries in the `confusedWith` list are limited to their Latin names, then we can do API lookups on them. We wouldn't be able to do a lookup on the `white forms...` sentence.